charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -122,10 +125,16 @@ spec:
                     values:
                       - fargate
       {{- end }}
-      {{- if $.Values.nodeSelector }}
       nodeSelector:
+        {{- if $.Values.nodeSelector }}
```

Comment:
I do not think that $ is needed here since we are not inside a loop

Reply:
also in the if next to it is the same

Reply:
Indeed, fixed!